libraries/tickfs/README.md

```diff
@@ -0,0 +1,93 @@
+# TickFS
+
+TickFS (Tiny Circular Key Value File System) is a small file system allowing
+key value pairs to be stored in Flash Memory.
+
+TickFS was written to allow the Tock OS kernel to persistently store app data
+on flash. It was written to be generic though, so other Rust applications can
+use it if they want.
+
+TickFS is based on similar concepts as
+[Yaffs1](https://yaffs.net/documents/how-yaffs-works]).
+
+TickFS provides ACID properties.
```

Comment:
How does it provide isolation? This implies a particular ordering of different clients. It would be good to explain how it provides each.

Reply:
It doesn't support concurrency so everything is isolated :)

I'll add some explanation and probably remove isolation.

Reply:
ACID semantics are usually defined in terms of transactions, so you need to state what a transaction is. A single read or write? If you define its transactions as single operations, and there are no concurrent operations, then yes, it has isolation. But this means, for example, that a client can't safely check for a key, and if it's not present then insert it.